src/site/content/en/media/video-and-source-tags/index.md

```diff
@@ -0,0 +1,361 @@
+</figure>
+
+{% CompareCaption %}
+A fallback poster makes it seem as if the first frame has been captured.
+{% endCompareCaption %}
+
+{% endCompare %}
+</div>
+
+### Ensure videos don't overflow containers
+
+When video elements are too big for the viewport, they may overflow their
+container, making it impossible for the user to see the content or use the
+controls.
+
+<div class="w-columns">
+  <figure class="w-figure">
+    <img src="./chrome-android-portrait-video-unstyled.png" alt="Android Chrome screenshot, portrait: unstyled video element overflows
+    viewport.">
+    <figcaption class="w-figcaption">Android Chrome screenshot, portrait: unstyled video element overflows
+    viewport.</figcaption>
+  </figure>
+  <figure class="w-figure">
+    <img src="./chrome-android-landscape-video-unstyled.png" alt="Android Chrome screenshot, landscape: unstyled video element overflows
+    viewport.">
+    <figcaption class="w-figcaption">Android Chrome screenshot, landscape: unstyled video element overflows
+    viewport.</figcaption>
+  </figure>
+</div>
+
+You can control video dimensions using JavaScript or CSS. JavaScript libraries
+and plugins such as [FitVids](http://fitvidsjs.com/) (outside the scope fo this
+article) make it possible to maintain appropriate size and aspect ratio, even
+for videos from YouTube and other sources.
+
+For simple uses like the ones I'm describing here, use [CSS media
+queries](/web/fundamentals/design-and-ux/responsive/#css-media-queries) to
+specify the size of elements depending on the viewport dimensions; `max-width:
+100%` is your friend.
+
+For media content in iframes (such as YouTube videos), try a responsive approach
+(like the one [proposed by John
+Surdakowski](http://avexdesigns.com/responsive-youtube-embed/)).
+
+{% Aside 'caution' %}
+Don't force element sizing that results in an [aspect
+ratio](https://www.google.com/search?q=aspect+ratio&oq=aspect+ratio&aqs=chrome..69i57j35i39j0l6.1896j0j7&sourceid=chrome&ie=UTF-8)
+different from the original video. Squashed or stretched looks bad.
+{% endAside %}
+
+**CSS:**
+
+```css
+.video-container {
+    position: relative;
+    padding-bottom: 56.25%;
+    padding-top: 0;
+    height: 0;
+    overflow: hidden;
+}
+
+.video-container iframe,
+.video-container object,
+.video-container embed {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+}
+```
+
+**HTML:**
+
+```html
+<div class="video-container">
+  <iframe src="//www.youtube.com/embed/l-BA9Ee2XuM"
+          frameborder="0" width="560" height="315">
+  </iframe>
+</div>
+```
+
+[Try it](https://googlesamples.github.io/web-fundamentals/fundamentals/media/responsive_embed.html)
+
+Compare the [responsive sample](https://googlesamples.github.io/web-fundamentals/fundamentals/media/responsive_embed.html)
+to the [unresponsive version](https://googlesamples.github.io/web-fundamentals/fundamentals/design-and-ux/responsive/unyt.html).
+
```

Comment:
Can you add a section mentioning picture-in-picture here? It can just be one paragraph linking to one of Francois's PiP posts.

Reply:
That requires JavaScript. We decided that all things JS were out of scope for phase I of this section.